### PR TITLE
Added the original source package name to Android data points

### PR DESCRIPTION
**Pull request by tylervz**
Now for example I can see `"com.misfitwearables.prometheus"` for step data coming from Misfit whereas before I saw nothing; `"source"` is not getting set for step data.

### CODE REVIEW

File: src/android/HealthPlugin.java [hunk 1]
```diff
@@ -408,7 +408,11 @@ private void query(final JSONArray args, final CallbackContext callbackContext)
                     obj.put("endDate", datapoint.getEndTime(TimeUnit.MILLISECONDS));
                     if (datapoint.getDataSource() != null)
                         obj.put("source", datapoint.getDataSource().getName());
-                    DataSource hhh = datapoint.getDataSource();
+                    if (datapoint.getOriginalDataSource() != null) {
+                        DataSource dataSource = datapoint.getOriginalDataSource();
+                        String appPkgName = dataSource.getAppPackageName();
+                        obj.put("originalSourcePackage", appPkgName);
```
Comment (tylervz) [inline src/android/HealthPlugin.java]: I would prefer this to be `obj.put("source", appPkgName);` and delete lines 409 & 410, but `datapoint.getDataSource().getName()` might be providing useful info for other kinds of data which I haven't tested.

**Comment by dariosalvi78** [comment]
Hi,

what's important to me is that the interface is the same in iOS and Android.
In iOS, thanks to the Telerik's plugin I am using, I can access both a "name" and a "bundleID", which are basically the same as "getDataSource().getName()" and "getDataSource().getAppPackageName()".

Let's do it like it is done in the original Telerik's plugin:
"source" is an object containing both a simple name and the complete package name, like:

```
source = {
  sourceName : "FitBit",
  sourceBundleId: "com.fitbit.fitbitapp"
}
```

or something like this.
Could you work on this improvement on the Android side? I will work the iOS part.

If you don't have time, or experience, I will do it myself, but, please, let me know.


Thanks!

**Comment by tylervz5** [comment]
Thanks @dariosalvi78, I can work on the Android side.

**Comment by tylervz** [comment]
Thanks @dariosalvi78. I can work on the Android side.

**Comment by dariosalvi78** [comment]
I have added the changes in iOS too.
In the end I decided to keep 2 separate fields sourceName and sourceBundleId within the root object instead of creating a sub-object. This is how it is done in Telerik's plugin too. Sorry for the confusion!
Let me know if it works.

**Pull request merged**